### PR TITLE
Website documentation for cog button states surf 774

### PR DESCRIPTION
**Pull request by araiford**
JIRA: https://jira.rax.io/browse/SURF-774

### CODE REVIEW

File: _component_design/actions-menu.md [hunk 1]
```diff
@@ -82,28 +76,28 @@ helix-ui-javascript: true
...
 {% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
 
-### Actions menu - simple
+### Specifications
 
-The most common uses of the actions menu are buttons labeled actions, split button, and cog.
+Section headers in the menu are optional. They are used if clarity is needed, or if there are more than seven actions in the list.
 
-Add a title in the action menus if it adds clarity. If there are more than seven actions in the list, the title is required.
+There are eight pixels of space between each section.
```
Comment (nimbinatus) [inline _component_design/actions-menu.md]: I think throughout the site we've been using the numerical form with the abbreviated unit for pixel measurements. If so, recommend s/`eight pixels`/`8px`.

File: _component_design/actions-menu.md [hunk 2]
```diff
@@ -118,18 +112,16 @@ The menu should span the length of the button. If an action name is longer than
 
 {% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
 
-### Actions menu - advanced
-
-The advanced actions menu is a longer list of actions separated by labels. The use of labels on long lists adds clarity and breaks the list into segments that are easier to parse.
+### Cog button states
 
-Use 8px of spacing between each section.
+When a cog is used on its own, it appears in gray 900 as a default state. Shown are the others states to consider in your designs.
```
Comment (nimbinatus) [inline _component_design/actions-menu.md]: s/`gray 900`/``` `gray900` ``` (essentially, written as preformatted text)

File: _component_design/actions-menu.md (hunk 1 above)
Comment (araiford) [reply _component_design/actions-menu.md]: Done

File: _component_design/actions-menu.md (hunk 2 above)
Comment (araiford) [reply _component_design/actions-menu.md]: dun too

File: _component_design/actions-menu.md [hunk 3]
```diff
@@ -118,18 +112,16 @@ The menu should span the length of the button. If an action name is longer than
 
 {% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
 
-### Actions menu - advanced
-
-The advanced actions menu is a longer list of actions separated by labels. The use of labels on long lists adds clarity and breaks the list into segments that are easier to parse.
+### Cog button states
 
-Use 8px of spacing between each section.
+When a cog is used on its own, it appears in gray900 as a default state. Shown are the others states to consider in your designs.
```
Comment (nimbinatus) [inline _component_design/actions-menu.md]: Needs code tics around `gray900`:
```
`gray 900`
```

**Pull request merged**